tsconfig-strict.json

```diff
@@ -143,5 +143,6 @@
     "extensions/interactions/MultipleChoiceInput/directives/multiple-choice-input-rules.service.ts",
     "extensions/interactions/MultipleChoiceInput/directives/multiple-choice-input-rules.service.spec.ts"
   ],
-  "include": ["typings/*.ts"]
+  "include": ["typings/*.ts"],
+  "exclude": ["extensions/classifiers/proto"]
```

Comment:
I think that this shouldn't be needed, could you try to remove this to see if it helps with the errors?